app/components/auth/firebase/lib/functions.js

```diff
@@ -42,6 +44,20 @@ export const createEmptyAuthUser = () => {
 }
 
 export const initAuth = () => {
+  let googleCreds = {
+    projectId: process.env.FIREBASE_PROJECT_ID || null,
+    clientEmail: process.env.FIREBASE_CLIENT_EMAIL || null,
+    // The private key must not be accessible on the client side.
+    privateKey: process.env.FIREBASE_PRIVATE_KEY || null,
+  }
+  if(typeof window === 'undefined' && process.env.GOOGLE_CREDS_PATH){
+    // firebase admin has to be initialized using service acccount json file.
+    const googleCredsFromFile = getGoogleCredsFromFile(process.env.GOOGLE_CREDS_PATH);
+    googleCreds.projectId = googleCredsFromFile.project_id;
+    googleCreds.privateKey = googleCredsFromFile.private_key;
+    googleCreds.clientEmail = googleCredsFromFile.client_email;
```

Comment:
Can we construct the googleCreds JSON directly within `getGoogleCredsFromFile.js` and return the ready object?

Reply:
Pushed the changes.